design-documents/graph-ql/coverage/negotiableQuotes.graphqls

```diff
@@ -1,205 +1,332 @@
 }
 
-type BundleRequisitionListItem implements NegotiableQuoteItemInterface
-@doc(description: "Negotiable Quote Item Implementation that for Bundle Products") {
-    id: ID! @doc(description: "Unique Identifier of Negotiable Quote Item")
-    product: ProductInterface!
-    qty: Float! @doc(description: "Quantity added")
-    customizable_options: [SelectedCustomizableOption] @doc(description: "custom Option selected")
-    bundle_options: [SelectedBundleOption]! @doc(description: "selected bundle options")
+type NegotiableQuoteComment {
+    id: ID!
+    created_at: String!
+    author: NegotiableQuoteUser!
+    creator_type: NegotiableQuoteCommentCreatorType!
+    value: String! @doc(description: "A simple (non-html) comment submitted by a seller or buyer")
 }
 
-type ConfigurableRequisitionListItem implements NegotiableQuoteItemInterface
-@doc(description: "Negotiable Quote Item Implementation that for Configurable Products") {
-    id: ID! @doc(description:  "Unique Identifier of Negotiable Quote Item")
-    product: ProductInterface!
-    qty: Float! @doc(description: "Quantity added")
-    customizable_options: [SelectedCustomizableOption] @doc(description: "custom Option selected")
-    configurable_options: [SelectedConfigurableOption] @doc(description: "Configurable options selected")
+enum NegotiableQuoteCommentCreatorType {
+    BUYER
+    SELLER
 }
 
-type AttachmentContent @doc(description: "Negotiable quote attachment file") {
-    base64_encoded_data: String!
-    mime_type: String!
+type NegotiableQuote {
+    id: ID!
     name: String!
+    items: [NegotiableQuoteItem!]
+    # Attachment Support is dependent on headless File Upload design
+    # attachments: [AttachmentContent]
+    comments: [NegotiableQuoteComment!]
+    history: [NegotiableQuoteHistoryEntry!]
+    prices: CartPrices
+    buyer: NegotiableQuoteUser!
+    created_at: String @doc(description: "Timestamp indicating when the negotiable quote was created.")
+    updated_at: String @doc(description: "Timestamp indicating when the negotiable quote was updated.")
+    status: NegotiableQuoteStatus!
 }
 
-type NegotiableQuoteComment {
-    id: ID!
-    parent_id: String!
-    author: String!
-    text: String
-    created_at: String
-    attachments: [String]
+# Implementation Note: Using the values from the "Buyer Status" column of the state mapping table in devdocs.
+# These states are identical between storefront/admin, but we name them differently for buyers.
+# https://devdocs.magento.com/guides/v2.4/b2b/negotiable-quote.html#quote-statuses
+enum NegotiableQuoteStatus {
+    SUBMITTED
+    PENDING
+    UPDATED
+    OPEN
+    ORDERED
+    CLOSED
+    DECLINED
+    EXPIRED
+}
+
+type NegotiableQuoteItem @doc(description: "A line item on a Negotiable Quote") {
+    id: ID! @doc(description: "ID of the line item (not product) in a Negotiable Quote")
+    item: CartItemInterface @doc(description: "Product assigned to line item, with selected configurations")
```

Comment:
`item` within Item is always not perfect...

Reply:
This is how it looks to query the item

```graphql
{
   negotiable_quote(id: 5) {
      items {
         id # line item ID
         item {
            id # product ID
            quantity
         }
      }
   }
}
```

I'm happy to change it, but I'm not sure what would be more descriptive here. 

- Considered `product`, but it's not _really_ a product you're getting, because that's in `CartItemInterface`
- Considered `line_item`, no strong feelings on this one either way
- Considered `cart_item`, but it's not quite a cart at this point, and didn't want to overload that term

Suggestions?